cmd/amp/commands.go

```diff
@@ -128,7 +129,7 @@ func addCommands(cmd *cobra.Command, c cli.Interface) {
 
 func info(c cli.Interface) error {
 	s := c.Server()
-	c.Console().Infof("[%s]\n", s)
+	fmt.Fprintf(c.Err(), "[%s]\n", s)
 
 	//// TODO: pToken.Claims panics
```

Comment:
@neha-viswanathan I know this isn't from you, but this big swath of commented out code really shouldn't be here. Since you're working with the file, would you mind opening an issue for this and then remove the code from here and paste it in the issue (and mention it came from here)? Thanks.